gilded-rose-llc-frontend/README.md

```diff
@@ -0,0 +1,51 @@
+# Gilded Rose LLC Frontend
+
+This project is built using Next.js, Typescript, and Tailwind CSS.
+
+## Information on topics related to this project
+
+- [React](https://reactjs.org/)
+- [TypeScript](https://www.typescriptlang.org/)
+- [TailwindCSS](https://tailwindcss.com/docs/guides/nextjs)
+- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/)
+- [Jest](https://jestjs.io/docs/getting-started)
+
+## Getting Started
+
+Clone the repo locally
+
+```
+git clone https://github.com/jramnani/GildedRoseJavaLLC.git
+cd GildedRoseJavaLLC
+cd gilded-rose-llc-frontend
+```
+
+Install dependencies
+
+```
+yarn install
+# or
+npm install
+```
+
+Run the development server:
+
+```
+npm run dev
+# or
+yarn dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+## Testing
+
+Tests are created using react-testing-library and jest. Unit tests are located in the `__tests__` directory.
+
+To run the tests:
+
+```
+yarn test
+# or
+npm test
```

Comment:
Same as the comment above.

Reply:
https://github.com/jramnani/GildedRoseJavaLLC/pull/27#discussion_r938173537

Reply:
Addressed above in [this comment](https://github.com/jramnani/GildedRoseJavaLLC/pull/27#discussion_r938338766)